docs(node-diagram): add overall node diagram for autoware.universe (#10)
tier4/autoware-documentation: docs/design/node-diagram.md

# Node Diagram

> The node diagram designs are depicted below for the Autoware.Core/Universe architecture.

Reviewer: How about this?

- Use active voice.
  - https://developers.google.com/style/voice?hl=en
- Autoware.Core/Universe -> Autoware Core/Universe
- Remove `below` because it's obvious.
- Remove `the`. (Warned by a tool.)

```suggestion
This page depicts the node diagram designs for Autoware Core/Universe architecture.
```

Author: Got it. Can I just commit this suggestion?

Reviewer: Sure, please click here.
![image](https://user-images.githubusercontent.com/31987104/157185226-64e33ef8-bdfd-4d73-9ff6-ab192b019b5b.png)

> ## Autoware.Core

Reviewer: ```suggestion
## Autoware Core
```

Author: done. https://github.com/tier4/autoware-documentation/pull/10/commits/f8c2577074c055c44ae080a15f91eed9af884832

TBD.

> ## Autoware.Universe

Reviewer: https://www.autoware.org/autoware

> The future of Autoware is in Autoware Core and Autoware Universe.  Autoware Core includes all functionality required to support the ODDs targeted by the Autoware project. 

```suggestion
## Autoware Universe
```

Author: done. https://github.com/tier4/autoware-documentation/pull/10/commits/f8c2577074c055c44ae080a15f91eed9af884832

![Node diagram](node-diagram/overall-node-diagram-autoware-universe.drawio.svg)

> [Open in draw.io for fullscreen](https://app.diagrams.net/#Htier4%2Fautoware-documentation%2Ftier4%2Fmain%2Fdocs%2Fdesign%2Fnode-diagram%2Foverall-node-diagram-autoware-universe.drawio.svg)

Reviewer: You are horrible.

```suggestion
[Open in draw.io for fullscreen](https://app.diagrams.net/?lightbox=1#Uhttps%3A%2F%2Ftier4.github.io%2Fautoware-documentation%2Ftier4%2Fmain%2Fdesign%2Fnode-diagram%2Foverall-node-diagram-autoware-universe.drawio.svg)
```

Reviewer: Or `latest` can be used.

Author: I'm horibe.

Reviewer: ```suggestion
[Open in draw.io for fullscreen](https://app.diagrams.net/?lightbox=1#Uhttps%3A%2F%2Ftier4.github.io%2Fautoware-documentation%2Flatest%2Fdesign%2Fnode-diagram%2Foverall-node-diagram-autoware-universe.drawio.svg)
```

> [Open in draw.io for fullscreen](https://app.diagrams.net/#Htier4%2Fautoware-documentation%2Fadd-node-topic-architecture%2Fdocs%2Fdesign%2Fnode-diagram%2Foverall-node-diagram-autoware-universe.drawio.svg)

Reviewer: I guess you need to fix here.

```suggestion
[Open in draw.io for fullscreen](https://app.diagrams.net/#Htier4%2Fautoware-documentation%2Ftier4/main%2Fdocs%2Fdesign%2Fnode-diagram%2Foverall-node-diagram-autoware-universe.drawio.svg)
```

Reviewer: (Perhaps `/` cannot be used.)

Author: done. https://github.com/tier4/autoware-documentation/pull/10/commits/c3ab3a8d35c1ff8b848673d7e4c3ed4e6c0b6d2f
